app/javascript/app.js

```diff
@@ -88,10 +88,27 @@ export default template({
     },
     sortStatements: function (statements) {
       return statements.sort((a, b) => {
+        const typeOrder = [
+          'verifiable',
+          'unverifiable',
+          'reconcilable',
+          'actionable',
+          'manually_actionable',
+          'done',
+          'reverted'
+        ]
         const namesA = parseFullName(a.person_name)
         const namesB = parseFullName(b.person_name)
-        const statementA = Object.assign({}, a, { firstName: namesA.first, lastName: namesA.last })
-        const statementB = Object.assign({}, b, { firstName: namesB.first, lastName: namesB.last })
+        const statementA = Object.assign({}, a, {
+          firstName: namesA.first,
+          lastName: namesA.last,
+          typeSort: typeOrder.indexOf(a.type)
```

Comment:
It's probably one of those optimizations that makes little difference when `n` is so small, but it jumps out at me a bit that there's a nested iteration here, so I'd make `typeOrder` an object that maps the type to an integer. It's up to you though :)

Reply:
In the spirit of optimising for humans and future changes I think I'm going to leave this. I've already had to change `typeOrder`, and it's slightly simpler reordering an array compared to reordering an object and then having to change the keys :)